.github/workflows/publish.yml

```diff
@@ -43,6 +43,8 @@ jobs:
       - name: Set dynamic environment variables
         run: |
           echo "SPINN_DIRS=$PWD/spinnaker_tools" >> $GITHUB_ENV
+      - name: Install pip, etc
+        uses: ./support/actions/python-tools
```

Comment:
Should also specify:
```
with:
    coverage: false
```
as we don't need support for handling coverage in this build.

Reply:
I got that by reading the source at https://github.com/SpiNNakerManchester/SupportScripts/blob/master/actions/python-tools/action.yml

Reply:
Otherwise, looks good; the right sort of fix.

Reply:
Done.